frontend/src/v5/ui/routes/viewer/drawings/drawingsList/drawingItem/drawingtem.styles.ts

```diff
@@ -0,0 +1,121 @@
+/**
+ *  Copyright (C) 2024 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import styled from 'styled-components';
+import { DrawingsCalibrationButton as DrawingsCalibrationButtonBase } from '@/v5/ui/routes/dashboard/projects/drawings/drawingsList/drawingsListItem/drawingsCalibrationButton/drawingsCalibrationButton.styles';
+import { CalibrationStates } from '@/v5/store/drawings/drawings.types';
+
+// TODO - fix after new palette is released
+export const Drawing = styled.div<{ $selected?: boolean }>`
+	position: relative;
+	cursor: pointer;
+	padding: 12px 14px 16px;
+	background-color: ${({ theme, $selected }) => ($selected ? '#edf0f8' : theme.palette.primary.contrast)};
+`;
+
+export const DrawingsCalibrationButton = styled(DrawingsCalibrationButtonBase).attrs({ width: 103 })<{ calibration: CalibrationStates }>`
+	.MuiButton-root {
+		border-radius: 5px;
+		border: solid 1px ${({ theme, calibration }) => calibration === CalibrationStates.CALIBRATED ? theme.palette.success.light : theme.palette.warning.light};
+		width: 103px;
+		height: 24px;
+	}
+`;
+
+export const MainBody = styled.div`
+	display: flex;
+	flex-direction: row;
+	gap: 10px;
+`;
+
+export const ImageContainer = styled.div`
+	border-radius: 5px;
+	border: solid 1px ${({ theme }) => theme.palette.base.lightest};
+	box-sizing: border-box;
+	height: 75px;
+	width: 75px;
+	overflow: hidden;
+`;
+
+export const InfoContainer = styled.div`
+	width: 235px;
+`;
+
+export const BreakingLine = styled.div`
+	padding-bottom: 5px;
+	line-height: 10px;
+`;
+
+export const BlueText = styled.span`
```

Comment:
Can we change "BlueText" for something more significant or at least that doesnt refer to a colour?